dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -45,6 +45,7 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
             // the org gets included in the search query.
             orgsToIncludeInSearch.put(ns.get(Constants.GIT_ORG), true);
         }
+        List<String> imagesThatCouldNotBeProcessed = new ArrayList<>();
```

Comment:
Can you also add an `AtomicInteger` counter of images successfully processed, so we can print a summary of number of images processed vs. failed. e.g. `11 of 18 (61%) parent images processed. 7 images failed to process, see error summary below.`

Reply:
For the error summary, let's create a POJO...
`List<ProcessingErrors> imagesThatCouldNotBeProcessed = new LinkedList<>();`
where `ProcesseingErrors` is: 
`ProcessingErrors(String imageNameAndTag, Optional<Exception> failure)`